semopenalex-app/data/templates/Template%3Ahttps%3A%2F%2Fsemopenalex.org%2Fclass%2FVenue.html

```diff
@@ -0,0 +1,260 @@
+            <{{page-resource}}> <https://semopenalex.org/property/countsByYear> ?countsByYear .
+            ?countsByYear <https://semopenalex.org/property/year> ?year ;
+                          <https://semopenalex.org/property/worksCount> ?publications
+          } ORDER BY ASC(?year)
+        " sets='[{"dataSetName": "Citations", "category": "year", "x": "year", "y": "publications"}]'
+          styles='[{
+            "provider": "highcharts",
+            "style": {
+              "series": [
+                {"type": "column", "showInLegend": false}
+              ]
+            }
+          }]'
+          style-highcharts='{
+            "legend": "hide",
+            "caption": {"align": "center", "text": "The last 10 years of publications."}
+          }'
+          dimensions='{"height": 180}'
+        >
+        </semantic-chart>
+      </div>
+      <div class="card venue-info">
+        <semantic-query query='
+          SELECT ?homepage ?issnL (group_concat(DISTINCT ?issn) as ?issns) WHERE {
+            BIND(<{{page-resource}}> as ?venue)
+              OPTIONAL {
+                ?venue <http://prismstandard.org/namespaces/basic/2.0/issn> ?issn .
+            }
+              OPTIONAL {
+                ?venue <http://xmlns.com/foaf/0.1/homepage> ?homepage .
+            }            
+              OPTIONAL {
+                ?venue <http://purl.org/spar/fabio/hasIssnL> ?issnL
+            }
+          }
+          GROUP BY ?homepage ?issnL
+        ' template='{{> tmp}}'>
+          <template id="tmp">
+            {{#each bindings}}
+              {{#if issnL}}
+                <div class="card__title">ISSN-L</div>
+                <div>{{issnL.value}}</div>
+              {{/if}}
+              {{#ifCond issns.value "!==" ""}}
+                <div class="card__title">ISSN</div>
+                <div>{{issns.value}}</div>
+              {{/ifCond}}
+                <div class="card__title">Home Page</div>
+                <div>
+                  {{#if homepage}}
+                    <a href="{{homepage.value}}">{{homepage.value}}</a>
+                  {{else}}
+                    <span>Home page not available.</span>
+                  {{/if}}
+                </div>
+            {{/each}}
+          </template>
+        </semantic-query>
+      </div>
+    </div>
+    <div class="concepts">
+      <semantic-query query='
+        SELECT DISTINCT ?concept WHERE {
+          BIND(<{{page-resource}}> as ?venue)
+          ?hostVenue <https://semopenalex.org/property/hasVenue> ?venue .
+          ?work <https://semopenalex.org/property/hasHostVenue> ?hostVenue .
+          ?work <https://semopenalex.org/property/hasConcept> ?concept  
+        }
+        LIMIT 20
+      ' template='{{> tmp}}'>
+        <template id="tmp">
+          {{#each bindings}}
+            <div class="chip">
+              <semantic-link class="chip__name" iri="{{concept.value}}"><mp-label iri='{{concept.value}}'></mp-label></semantic-link>
```

Comment:
you do not need two components fetching labels for given `IRI`. 
please remove `mp-label` component.